kubernetes-deploy.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.3.0'
   spec.add_dependency "activesupport", ">= 4.2"
-  spec.add_dependency "kubeclient", "~> 2.3"
+  spec.add_dependency "kubeclient", "~> 2.4"
```

Comment:
Looks like not much changed: https://github.com/abonas/kubeclient/compare/v2.3.0...v2.4.0

Reply:
Yeah, I was hoping they might have released a version with your NotFound error class. That's not the case, but I figured we might as well update anyway.